src/utils/decorators/input/input.scss

```diff
@@ -195,7 +195,7 @@
   height: 15px;
   position: absolute;
   right: 5px;
-  top: 6px;
+  top: 28px;
```

Comment:
It's not really a fix to change fixed pixel, fixed positioning from one value to another, as this will push it out in some other context (now, or in the future)

I suggest using flex and fixing this once and for all :-P